src/entries/popup/pages/home/Tokens.tsx

```diff
@@ -203,10 +207,12 @@ export function Tokens() {
         <Box style={{ overflow: 'auto' }}>
           {assetsRowVirtualizer.getVirtualItems().map((virtualItem) => {
             const { key, size, start, index } = virtualItem;
-            const token = allAssets[index];
+            const token = filteredAssets[index];
+            const pinned = uniqueIds.some((id) => id === token.uniqueId);
+
             return (
               <Box
-                key={key}
+                key={`${token.uniqueId}-${key}`}
```

Comment:
When pressed the react state will reset because key changes. This is useful so that we're no longer in a "pressed" active state.